plugins/action_update_review_status/src/updateReviewsStatuses.ts

```diff
@@ -24,7 +24,8 @@ async function updateReviewsStatuses({
 
   const { changedApplicationResponses = [] } = parameters
   const stageId = parameters?.stageId || applicationData?.stageId
-  const level = parameters?.level || applicationData?.reviewData?.levelNumber || 1
+  const currentReviewLevel = applicationData?.reviewData?.levelNumber
+  const level = parameters?.level || currentReviewLevel ? currentReviewLevel + 1 : 1
```

Comment:
Still throwing a Typescript error when running the `database_init`.

See my current solution to have it running:
```suggestion
  const currentReviewLevel = parameters?.level || applicationData?.reviewData?.levelNumber
  const level = currentReviewLevel ? currentReviewLevel + 1 : 1
```

Reply:
I've made the change, but to something else